healthcare/api-client/dicom/dicom_stores.py

```diff
@@ -25,22 +25,23 @@ def get_client(service_account_json, api_key):
     """Returns an authorized API client by discovering the Healthcare API and
     creating a service object using the service account credentials JSON."""
     api_scopes = ['https://www.googleapis.com/auth/cloud-platform']
-    api_version = 'v1alpha'
+    api_version = 'v1alpha2'
```

Comment:
Just FYI, I was seeing v1beta1 elsewhere, maybe a typo?

Reply:
This is a typo, a serious one. I'll fix it after rerunning tests.

Reply:
Checking again if this should be alpha or beta